Formula/stone-soup.rb

```diff
@@ -23,26 +21,19 @@ class StoneSoup < Formula
 
   depends_on "pkg-config" => :build
   depends_on "python@3.10" => :build
+  depends_on "pyyaml" => :build
   depends_on "lua@5.1"
   depends_on "pcre"
   depends_on "sqlite"
 
   fails_with gcc: "5"
 
-  resource "PyYAML" do
-    url "https://files.pythonhosted.org/packages/36/2b/61d51a2c4f25ef062ae3f74576b01638bebad5e045f747ff12643df63844/PyYAML-6.0.tar.gz"
-    sha256 "68fb519c14306fec9720a2a5b45bc9f0c8d1b9c72adf45c37baedfcd949c35a2"
-  end
-
   def install
     ENV.cxx11
     ENV.prepend_path "PATH", Formula["python@3.10"].opt_libexec/"bin"
     python3 = "python3.10"
     ENV.prepend_create_path "PYTHONPATH", buildpath/"vendor"/Language::Python.site_packages(python3)
```

Comment:
Should we also get rid of all this?  Does this even need to depend on `python@3.10` at all anymore?

Reply:
agree, maybe line 39 to line 41 can be removed

Reply:
I can give a try in my local

Reply:
good to be removed